web/sites/default/config/views.view.group_nodes.yml

```diff
@@ -1270,7 +1270,7 @@ display:
           exclude: false
           alter:
             alter_text: true
-            text: "{% if moderation_state == \"Published\" %}\r\n<span class=\"badge badge-success\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% elseif moderation_state == \"In review\" %}\r\n<span class=\"badge badge-warning\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{{ field_reviewer }}\r\n{% elseif moderation_state == \"Unpublished/Archived\" %}\r\n<span class=\"badge badge-danger\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% elseif moderation_state == \"Draft\" %}\r\n<span class=\"badge badge-warning\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% else %}\r\n<span class=\"badge badge-light\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% endif %}\r\n{{ is_locked }}"
+            text: "{% if moderation_state == \"Published\" %}\r\n<span class=\"badge cloudy-u-color-background-green-100 cloudy-u-color-text-green-500  cloudy-u-color-border-green-500 cloudy-u-border-1 cloudy-u-border-solid\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% elseif moderation_state == \"In review\" %}\r\n<span class=\"badge cloudy-u-color-background-yellow-100 cloudy-u-color-text-yellow-800  cloudy-u-color-border-yellow-800 cloudy-u-border-1 cloudy-u-border-solid\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{{ field_reviewer }}\r\n{% elseif moderation_state == \"Unpublished/Archived\" %}\r\n<span class=\"badge cloudy-u-color-background-red-100 cloudy-u-color-text-red-500 cloudy-u-color-border-red-500 cloudy-u-border-1 cloudy-u-border-solid\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% elseif moderation_state == \"Draft\" %}\r\n<span class=\"badge cloudy-u-color-background-orange-100 cloudy-u-color-text-orange-500  cloudy-u-color-border-orange-500 cloudy-u-border-1 cloudy-u-border-solid\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% else %}\r\n<span class=\"badge cloudy-u-color-background-orange-100 cloudy-u-color-text-orange-500 cloudy-u-color-border-orange-500 cloudy-u-border-1 cloudy-u-border-solid\" title=\"{{ moderation_state }}\" data-toggle=\"tooltip\">{{ moderation_state }}</span>\r\n{% endif %}\r\n{{ is_locked }}"
```

Comment:
@brittanysmart , I think this line is breaking the group manage content view's ability to display anything other than published or unpublished. I can't get it to show draft or in review anymore. Can you take a look: https://powr-2133-portlandor.pantheonsite.io/wheeler/news/2020/3/13/multnomah-county-libraries-closed-until-further-notice

Reply:
This should be fixed now.